Log2SubBand/src/menuUI/InputMenu.form

```diff
@@ -0,0 +1,728 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
JFormDesigner 5.2 generated this file -> this file can be removed once input menu UI is finalised.